lib/chrome-driver.js

```diff
@@ -24,7 +23,7 @@ ChromeDriver.prototype.start = function () {
     cwd: process.cwd(),
     env: this.getEnvironment()
   }
-  this.process = ChildProcess.spawn(process.execPath, args, options)
+  this.process = ChildProcess.spawn(this.path, args, options)
```

Comment:
So it turns out the setTimeout loop that I "fixed" in this PR wasn't working for me because the atom jasmine helper will spy on and mock setTimeouts.  Setting that environment variable does seem to allow process.execPath to work, but execution still ends up getting to the client.init() call, and then never resolve or reject.